src/cares_wrap.cc

```diff
@@ -318,7 +318,7 @@ void cares_wrap_hostent_cpy(struct hostent* dest, struct hostent* src) {
 
   /* copy `h_name` */
   size_t name_size = strlen(src->h_name) + 1;
-  dest->h_name = node::Malloc<char>(name_size);
+  dest->h_name = reinterpret_cast<char*>(node::Malloc(name_size));
```

Comment:
@XadillaX Yes, you’ll just need to try and see how to resolve the conflicts that you’re going to get. I suspect that will be a bit of work, so only do that if you have the time; for now, just changing to `static_cast` here is definitely easier. :)